src/display.c

```diff
@@ -339,8 +339,16 @@ PyImaging_GrabScreenWin32(PyObject* self, PyObject* args)
     screen = CreateDC("DISPLAY", NULL, NULL, NULL);
     screen_copy = CreateCompatibleDC(screen);
 
-    width = GetDeviceCaps(screen, HORZRES);
-    height = GetDeviceCaps(screen, VERTRES);
+    if (multimonitor) {
+        x = GetSystemMetrics(SM_XVIRTUALSCREEN);
+        y = GetSystemMetrics(SM_YVIRTUALSCREEN);
+        width = GetSystemMetrics(SM_CXVIRTUALSCREEN);
+        height = GetSystemMetrics(SM_CYVIRTUALSCREEN);
+    } else {
+        // FIXME doesn't work with DPI scaling
+        width = GetSystemMetrics(SM_CXSCREEN);
+        height = GetSystemMetrics(SM_CYSCREEN);
```

Comment:
Changing from `GetDeviceCaps(screen, HORZRES)` to `GetSystemMetrics(SM_CXSCREEN)` is separate to the 'multimonitor' feature. Could you talk about why you've done this?

Reply:
I was hoping it would resolve the DPI issue, but it seems they are equivalent. So I suppose there is no reason for this change.